extensions/stackdriver/log/logger.cc

```diff
@@ -185,6 +185,13 @@ void Logger::fillAndFlushLogEntry(
   (*label_map)["protocol"] = request_info.request_protocol;
   (*label_map)["log_sampled"] = request_info.log_sampled ? "true" : "false";
   (*label_map)["connection_id"] = std::to_string(request_info.connection_id);
+  (*label_map)["route_name"] = request_info.route_name;
+  (*label_map)["upstream_host"] = request_info.upstream_host;
+  (*label_map)["upstream_cluster"] = request_info.upstream_cluster;
+  (*label_map)["requested_server_name"] = request_info.request_serever_name;
+  (*label_map)["x-envoy-original-path"] = request_info.x_envoy_original_path;
```

Comment:
Are sd log entries searchable by these labels ?

Reply:
yes.. it would be something like: labels.x-envoy-original-path